app/retail/templates/activity.html

```diff
@@ -18,7 +18,8 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta name="title" content="Activity Feed | Gitcoin">
```

Comment:
`<meta >` doesn't require element end tags/closures.  `<meta />` is technically valid HTML5 because it doesn't disallow it, but doesn't validate as proper HTML4.

Reply:
So, should I remove `/` from all of them or keep them?

Reply:
I think we might as well remove them since they're superfluous, if not invalid.  Other than that, looks good so far.